src/dependency_decisions_overwrites.yml

```diff
@@ -34,3 +34,45 @@
     :why: https://raw.githubusercontent.com/chaychoong/coveragepy-lcov/master/LICENSE
     :versions: []
     :when: 2019-08-30 05:45:35.317663000 Z
+- - :license
+  - libcurl
+  - CURL
+  - :who: BjoernAtBosch <Bjoern.Hornburg@de.bosch.com>
+    :why: https://curl.se/docs/copyright.html
+    :versions: []
+    :when: 2022-08-30 16:12:04.826121174 Z
+- - :license
+  - openssl
+  - OpenSSL License AND SSLeay License
```

Comment:
Need a version specifier here. OpenSSL >=3.0.0 is Apache v2 according to the license text

Reply:
I tried that, but the version entry is ignored (as long as you stick to the YML format) - I also looked into the code once more and it confirmed this experience.
Yet another bug/limitation of the Pivotal License Finder ...

Reply:
I've removed them (for now), to be added to the respective using repository